package.json

```diff
@@ -1,84 +1,90 @@
+  "jest": {
+    "preset": "ts-jest",
+    "verbose": true,
+    "clearMocks": true,
+    "testEnvironment": "node",
+    "moduleFileExtensions": [
+      "js",
+      "ts"
+    ],
+    "testMatch": [
+      "**/*.test.ts"
+    ],
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "/dist/"
+    ],
+    "transform": {
+      "^.+\\.ts$": "ts-jest"
     },
-    "bugs": {
-      "url": "https://github.com/doradsoft/html-flip-book/issues"
-    },
-    "engines": {
-        "node": ">=20"
-    },
-    "scripts": {
-        "bundle": "npm run format:write && npm run package",
-        "ci-test": "npx jest",
-        "coverage": "npx make-coverage-badge --output-path ./badges/coverage.svg",
-        "format:write": "npx prettier --write .",
-        "format:check": "npx prettier --check .",
-        "lint": "npx eslint . -c ./.github/linters/.eslintrc.yml",
-        "package": "npx ncc build src/index.ts -o dist --source-map --license licenses.txt",
-        "package:watch": "npm run package -- --watch",
-        "test": "npx jest",
-        "all": "npm run format:write && npm run lint && npm run test && npm run coverage && npm run package"
-    },
-    "author": "DoradSoft",
-    "license": "MIT",
-    "jest": {
-        "preset": "ts-jest",
-        "verbose": true,
-        "clearMocks": true,
-        "testEnvironment": "node",
-        "moduleFileExtensions": [
-          "js",
-          "ts"
-        ],
-        "testMatch": [
-          "**/*.test.ts"
-        ],
-        "testPathIgnorePatterns": [
-          "/node_modules/",
-          "/dist/"
-        ],
-        "transform": {
-          "^.+\\.ts$": "ts-jest"
-        },
-        "coverageReporters": [
-          "json-summary",
-          "text",
-          "lcov"
-        ],
-        "collectCoverage": true,
-        "collectCoverageFrom": [
-          "./src/**"
-        ]
-    },
-    "dependencies": {
-        "@actions/core": "^1.10.1"
-    },
-    "devDependencies": {
-        "@types/node": "^20.14.2",
-        "@typescript-eslint/eslint-plugin": "^8.0.0-alpha.14",
-        "@typescript-eslint/parser": "^8.0.0-alpha.14",
-        "css-loader": "^7.1.2",
-        "eslint": "^9.4.0",
-        "eslint-plugin-github": "^5.0.1",
-        "eslint-plugin-jsonc": "^2.16.0",
-        "eslint-plugin-prettier": "^5.1.3",
-        "make-coverage-badge": "^1.2.0",
-        "mini-css-extract-plugin": "^2.9.0",
-        "prettier": "^3.3.1",
-        "prettier-eslint": "^16.3.0",
-        "sass": "^1.77.4",
-        "sass-loader": "^14.2.1",
-        "style-loader": "^4.0.0",
-        "ts-loader": "^9.5.1",
-        "ts-node": "^10.9.2",
-        "typescript": "^5.4.5",
-        "webpack": "^5.91.0",
-        "webpack-cli": "^5.1.4"
-    }
+    "coverageReporters": [
+      "json-summary",
+      "text",
+      "lcov"
+    ],
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "./src/**"
+    ]
+  },
+  "dependencies": {
+    "@actions/core": "^1.10.1"
```

Comment:
This should be a devDependency `@actions/core` as it's not going to be put in product for sure.